charts/all-in-one/templates/configmap-rudolf-currency-notifier.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if $.Values.rudolfCurrencyNotifier.enabled }}
```

Comment:
Could you add a default `.Values.rudolfCurrencyNotifier.enabled`? (maybe false)

You can add the below lines into the bottom of `charts/all-in-one/values.yaml` file.

```yaml
rudolfCurrencyNotifier:
  enabled: false
```